rudder-agent/SOURCES/patches/UBUNTU_10_04/0001-use-bundled-openssl.patch

```diff
@@ -42,3 +42,12 @@ diff -Naur debian/rules debian/rules
  	dh_install --SOURCEDIR=$(CURDIR)/SOURCES/ rudder.conf /etc/ld.so.conf.d
  	# Install a verification script for cron
  	dh_install --SOURCEDIR=$(CURDIR)/SOURCES/ check-rudder-agent /opt/rudder/bin/
+@@ -107,7 +111,7 @@
+ #	dh_perl
+ 	dh_makeshlibs
+ 	dh_installdeb
+-	dh_shlibdeps
++#	dh_shlibdeps
```

Comment:
This should work for all dpkg build.
We must try to avoid specific cases as much as possible. 
Can you do this on the original file instead ?